### ISSUE TITLE
Document behavior for various facts

### ISSUE THREAD

**Issue by lzap** [theforeman/foreman-documentation#124]
Foreman performs some actions for some facts, according to its configuration. This can be often confusing for users, let's document it somewhere.

| Fact provider | Fact name | Description | Configurable |
| --- | --- | --- | --- |
| Puppet | environment | Updates Host's Puppet environment from fact overwriting its previous value | Administer - Setting - Puppet - Update environment from facts |

This issue is WIP.

**Comment by maximiliankolb** [comment]
context: provisioning a host, updating it, facts get updated such as OS minor version.

story: excluding updating certain fields when uploading fact.

**Comment by maximiliankolb** [comment]
Yes, I agree. In the past, I thought that I might get to it eventually but never really had the time/muse.

Considering that foreman_puppet has been around for years and nobody ever requrested any of this information, I lean towards "close with won't fix".
